src/components/board/Card.jsx

```diff
@@ -1,7 +1,45 @@
 import React from "react";
+import Image from "next/image";
+import Judge from "../../../public/images/judge.png";
+import { GrMail } from "react-icons/gr";
+import { FaLinkedinIn } from "react-icons/fa";
+import Link from "next/link";
 
-const Card = () => {
-  return <div>Card</div>;
+const Card = ({ name, title, year, major, pronouns, linkedIn, email }) => {
+  return (
+    <div className="font-poppins m-4 bg-white w-64 h-96 rounded-3xl shadow flex flex-col items-center justify-center">
+      <div className="w-56 h-5/6 flex flex-col justify-center items-start">
+        <div className="flex w-full h-48">
```

Comment:
remove this div, add width and height to the image class. Don't do h-48, use fraction